swap/swap_test.go

```diff
@@ -1795,9 +1795,81 @@ func TestSwapLogToFile(t *testing.T) {
 		t.Fatal(err)
 	}
 	logString := string(b)
+
 	if !strings.Contains(logString, "sending cheque") {
 		t.Fatalf("expected the log to contain \"sending cheque\"")
 	}
+
+}
+
+func TestSwapActions(t *testing.T) {
```

Comment:
it's great that you added this, but try to extract the duplicated code between this function and the original `TestSwapLogToFile` test

Reply:
They are similar, but not necessarily the same.
I would refrain from refactoring until it's used in more logging tests.

Reply:
yes, at this point only the first lines are the same.